CHANGELOG.md

```diff
@@ -36,13 +36,13 @@ This section is for maintaining a changelog for all breaking changes for the cli
 ### Added
 - Added support for icu_collation_keyword type ([#725](https://github.com/opensearch-project/opensearch-java/pull/725))
 - Added support for flat_object field property ([#735](https://github.com/opensearch-project/opensearch-java/pull/735))
-
+- Added toQuery method in Query and QueryVariant ([#760](https://github.com/opensearch-project/opensearch-java/pull/760)
 ### Dependencies
 
 ### Changed
 
 ### Deprecated
-
+- deprecated _toQuery() in Query and QueryVariant ([#760](https://github.com/opensearch-project/opensearch-java/pull/760)
```

Comment:
Capitalize `Deprecated` and maybe quote `_toQuery` and other types (feels like we should be doing it, nbd).